gpu/gpu.go

```diff
@@ -28,6 +28,9 @@ type handles struct {
 var gpuMutex sync.Mutex
 var gpuHandles *handles = nil
 
+// TODO verify this is the correct min version
+const CudaComputeMajorMin = 5
```

Comment:
I haven't been able to find a definitive answer for what the minimum version is.  I know 3.0 is too old, and 7.5 works based on the GPUs I've got.  Looking at the cuBLAS API docs, I think 5.0 is plausible, but it's ambiguous.